packages/lockfile-lint-api/src/validators/ValidateHost.js

```diff
@@ -1,16 +1,16 @@
 'use strict'
 
-const {URL} = require('url')
-const debug = require('debug')('lockfile-lint-api')
 const {REGISTRY} = require('../common/constants')
 
+const noop = () => {}
 module.exports = class ValidateHost {
-  constructor ({packages} = {}) {
+  constructor ({packages, debug = noop} = {}) {
```

Comment:
`noop` here?

Reply:
noop by default, debug if passed.

Reply:
Ok my bad. I missed that you have actually defined it here `const noop = () => {}` a few lines before :-)